README.md

```diff
@@ -31,6 +32,55 @@ Theoretically, Pieman can be run on any GNU/Linux, however, it was very carefull
 
 ### Installation
 
+#### Docker
+
+First, get the latest Pieman Docker image from Docker Hub.
+
+```
+$ docker pull cusdeb/pieman
+```
+
+Then, get `docker-pieman.sh`
+
+```
+$ wget https://raw.githubusercontent.com/tolstoyevsky/pieman/master/docker-pieman.sh -O docker-pieman.sh
+```
+
+or using `curl`
+
+```
+$ curl -o docker-pieman.sh https://raw.githubusercontent.com/tolstoyevsky/pieman/master/docker-pieman.sh
+```
+
+Note, that the script requires bash 4 and higher, so macOS users should upgrade their bash if they haven't done it yet.
+
+##### Usage
+
+Simply run `docker-pieman.sh` to create an image based on Raspbian Stretch for Raspberry Pi 3.
+
+```
+$ chmod +x docker-pieman.sh
+$ ./docker-pieman.sh
+```
+
+Under the hood the script runs
+
+```
+$ docker run --privileged --rm -v $(pwd):/result -v /dev:/dev cusdeb/pieman
+```
+
+It's quite wordy, isn't it? `docker-pieman.sh` is intended to hide the details and provide an easy-to-use command-line interface.
+
+Another example shows how to create an image based on Ubuntu Xenial for Raspberry Pi 2 with [htop](https://packages.debian.org/stretch/htop) and [mc](https://packages.debian.org/stretch/mc) pre-installed
+
+```
+$ ./docker-pieman.sh -e DEVICE=rpi-2-b -e OS=ubuntu-xenial-armhf -e INCLUDES=htop,mc
+```
+
+The built images will be located in the current directory. By the way, you can specify the name of your project via the `PROJECT_NAME` environment variable. You can find details on `DEVICE`, `INCLUDES`, `OS`, `PROJECT_NAME` and other environment variables (called parameters) which help customizing images in the Documentation section.
```

Comment:
What if we add PROJECT_NAME to the example above? I prefer "The built images will be located in `build/<project name>`" more than "The built images will be located in the current directory.". One more point, when I read the paragraph at the first time it wasn't obvious for me that PROJECT_NAME means filename of the output file.

Reply:
1. We can make the Pieman Docker container put the images into `build/<project name>`. I guess it would be more logical if the behavior of the script was the same as the behavior of the Docker container.
2. We can make Pieman show something like

> The image `image name` is ready.

when Pieman is done.